IMASampleApp/app/src/main/java/com/ooyala/sample/players/CustomConfiguredIMAPlayerActivity.java

```diff
@@ -32,6 +33,9 @@ public final static String getName() {
   final String PCODE  = "R2d3I6s06RyB712DN0_2GsQS-R-Y";
   final String DOMAIN = "http://ooyala.com";
 
+  // Write the sdk events text along with events count to log file in sdcard if the log file already exists
+  SDCardLogcatOoyalaEventsLogger playbacklog = new SDCardLogcatOoyalaEventsLogger();
```

Comment:
Made the changes

On Mon, Mar 14, 2016 at 5:00 PM, jonslenkooyala <notifications@github.com>
wrote:

> In
> IMASampleApp/app/src/main/java/com/ooyala/sample/players/CustomConfiguredIMAPlayerActivity.java
> <https://github.com/ooyala/android-sample-apps/pull/79#discussion_r56096937>
> :
>
> > @@ -32,6 +33,9 @@ public final static String getName() {
> >    final String PCODE  = "R2d3I6s06RyB712DN0_2GsQS-R-Y";
> >    final String DOMAIN = "http://ooyala.com";
> >
> > +  // Write the sdk events text along with events count to log file in sdcard if the log file already exists
> > +  SDCardLogcatOoyalaEventsLogger playbacklog = new SDCardLogcatOoyalaEventsLogger();
>
> right, yes, :-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/ooyala/android-sample-apps/pull/79/files#r56096937
>



-- 
Regards,

BinduMadhav Sondur
310, Crescent Village Circle , #2110 ,
San Jose , CA - 95134

PH: 315-247-5338